audit_history/mixins.py

```diff
@@ -38,6 +42,7 @@ def _create_history_entry(modification_time, current_user, event, **payload):
                      'name': current_user.get_full_name(),
                      'is_staff': current_user.is_staff
                  }}
+        payload = json.loads(json.dumps(payload, default=json_formatter))
```

Comment:
I'm not sure I fully understand this. It's dumping it and then immediately loading it again? Basically converting datetimes into strings, for example?
Is there maybe a cleaner way to express this?

Reply:
Payload can have any data that we can imagine. So in my case we ensure that when we save this data we don't receive TypeError: datetime.datetime(2019, 1, 30, 11, 56, 33, 369765, tzinfo=<UTC>) is not JSON serializable orTypeError when we have Decimal in payload